### ISSUE TITLE
[MDN] Date format from “Payment confirmed” email has an additional space

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#13084]
**FxA version:**

* 1.233.1;

**Prerequisites:**

* Be localized in Slovenia;

**Steps to reproduce:**

1. Access[ |https://developer.allizom.org/en-US/plus#subscribe][+https://developer.allizom.org/plus#subscribe+](https://developer.allizom.org/en-US/plus#subscribe)  ;
1. Make a subscription for MDN;
1. Open the “Payment confirmed” email;
1. Observe the date format;

**Expected result:**

* Date format has no additional space;

**Actual result:**

* Date has an additional space displayed;

**Notes:**

* See the attached screenshot;

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXA-5215)
┆Attachments: <a href="https://mozilla-hub.atlassian.net/rest/api/2/attachment/content/24329">image-20220526-141111.png</a> | <a href="https://mozilla-hub.atlassian.net/rest/api/2/attachment/content/24386">image-20220527-141702.png</a> | <a href="https://mozilla-hub.atlassian.net/rest/api/2/attachment/content/24325">Screenshot 2022-05-26 154257.png</a>

**Comment by data-sync-user** [comment]
➤ Maria Berlinger commented:

The same issue is reproducible for the cancelled subscription email.

See the attached screenshot.

!image-20220526-141111.png|width=1228,height=597!

**Comment by data-sync-user** [comment]
➤ Luminita Buruian commented:

For Latvia, there is an extra dot at the end of the date.

!image-20220527-141702.png|width=1399,height=475!

**Comment by data-sync-user** [comment]
➤ Meghan Sardesai commented:

I took a look and here's what I found:

  * Latvia extra period: still a mystery. I can't find default text anywhere that has an ending period (the last commits to that line were 2-5 months ago that don't appear to include it (unless I missed one)). It's not included in Latvia's *.ftl . Further, Latvia has a low progress rate for translations in Pontoon.
  * Added spaces in date for Slovenia: although we had l10n changes in the latest release, the closest change I could find, documented in the CHANGELOG ( https://github.com/mozilla/fxa/compare/v1.231.4...v1.232.0#diff-39d3d5fe0ef6f45ae088965fcd6958fbf7baa09dda0c8219c023a6e48d9a2fca ) on line 15 is from commit 44c8823f8 ( https://github.com/mozilla/fxa/commit/44c8823f8 ), but it appears to only update nextInvoiceDateOnly and the space is in both. This could have been an existing issue and how the local date is constructed and/or how it's stringified.
